birdhouse/components/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Old paths from deprecated components. Keep these so that old config files remain uncommittable after updates.
+portainer/config/proxy/conf.extra-service.d/portainer.conf
```

Comment:
Should `components/portainer` be added as well?

Reply:
That's not necessary really. All the other files in the portainer directory are tracked by git so they will be deleted by this change. Only the file that was in the .gitignore will be left behind.

Reply:
I'm thinking in terms of any place that could have them still present on this could re-inject them by mistake. Though, not blocking, since this could be detected and reverted if it happens in another PR.